client/landing/stepper/declarative-flow/flows/site-migration-flow/test/site-migration-flow.tsx

```diff
@@ -75,6 +79,21 @@ describe( 'Site Migration Flow', () => {
 		jest.restoreAllMocks();
 	} );
 
+	describe( 'isSignupFlow', () => {
+		afterEach( () => {
+			window.location.search = '';
+		} );
+
+		it( 'returns true when there is no siteSlug or siteId on query params', () => {
+			expect( siteMigrationFlow.isSignupFlow ).toBe( true );
+		} );
+
+		it( 'returns false when there is siteSlug or siteId on query params', () => {
+			window.location.search = '?siteSlug=123';
+			expect( siteMigrationFlow.isSignupFlow ).toBe( false );
+		} );
```

Comment:
Might be worth adding an explicit check for `siteId` as well, just to be extra-thorough.